sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/xml/StaxXmlGenerator.scala

```diff
@@ -29,10 +31,11 @@ import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.util.{ArrayData, DateFormatter, DateTimeUtils, MapData, TimestampFormatter}
 import org.apache.spark.sql.catalyst.util.LegacyDateFormats.FAST_DATE_FORMAT
 import org.apache.spark.sql.types._
-import org.apache.spark.unsafe.types.UTF8String
+import org.apache.spark.types.variant.VariantUtil._
```

Comment:
Lets use VariantUtil prefix in the code. Its easier to see where the function is coming from
```suggestion
import org.apache.spark.types.variant.VariantUtil
```

Reply:
I imported all entities in `VariantUtil` to avoid calling it tens of times in this class. Let me do that instead, but I just want to call out the tradeoff